deepchecks/nlp/checks/model_evaluation/weak_segments_performance.py

```diff
@@ -102,11 +119,10 @@ def run_logic(self, context: Context, dataset_kind) -> CheckResult:
             loss_per_sample = [log_loss([y_true], [y_proba], labels=sorted(context.model_classes)) for y_true, y_proba
                                in zip(list(text_data.label), proba_values)]
 
-        additional_data = select_from_dataframe(text_data.additional_data, self.columns, self.ignore_columns)
-        if additional_data.shape[1] < 2:
```

Comment:
actually I don't know why. I mean, 1D segmentation is more boring but can still be useful.

Reply:
What?

Reply:
why won't we allow features.shape[1] == 1